File: Initial Prototype/Client/MyApplicationProject/MyApplication/src/main/java/com/example/myapplication/dtos/NotificationMarkerDTO.java
package com.example.myapplication.dtos;

/**
 * DTO object used to mark a notification as delivered.
 */
public class NotificationMarkerDTO {
    private int NotificationId;
    private int UserId;

    public NotificationMarkerDTO(int userId, int notificationId) {
        this.UserId = userId;
        this.NotificationId = notificationId;
    }
}

File: Initial Prototype/ServerSide/FindNDrive/FindNDriveServices2/DTOs/NotificationMarkerDTO.cs
// --------------------------------------------------------------------------------------------------------------------
// <copyright file="NotificationMarkerDTO.cs" company="">
//   
// </copyright>
// <summary>
//   The notification marker dto.
// </summary>
// --------------------------------------------------------------------------------------------------------------------

namespace FindNDriveServices2.DTOs
{
    using System.Runtime.Serialization;

    /// <summary>
    /// The notification marker dto.
    /// </summary>
    [DataContract]
    public class NotificationMarkerDTO
    {
        /// <summary>
        /// Gets or sets the notification id.
        /// </summary>
        [DataMember]
        public int NotificationId { get; set; }

        /// <summary>
        /// Gets or sets the user id.
        /// </summary>
        [DataMember]
        public int UserId { get; set; }
    }
}